arch/x86/core/ia32/intstub.S

```diff
@@ -13,6 +13,8 @@
  * exception) stubs are implemented in this module.  The stubs are invoked when
  * entering and exiting a C interrupt handler.
  */
+#define DT_DRV_COMPAT intel_loapic
+#define LOAPIC_BASE_ADDRESS DT_REG_ADDR(DT_DRV_INST(0))
```

Comment:
Is the shorter form `DT_INST_REG_ADDR(0)`? 
BTW, do we need this COMPAT and instance identifiers? We can do it with `DT_NODELABEL` like it was done for watchdog 
@jhedberg

Reply:
OK, we decided to use DT_INST API, but you can still use shorter form

Reply:
Hi @finikorg ,
Thanks for the suggestion. I have fixed this in my latest commit, especially for the non-driver code.